pkg/unserializer/unserializer_cdx.go

```diff
@@ -226,3 +221,29 @@ func (u *UnserializerCDX14) licenseChoicesToLicenseString(lcs *cdx.Licenses) str
 	}
 	return s
 }
+
+// phaseToSBOMType converts a CycloneDX lifecycle phase to an SBOM document type
+// note that most of the CycloneDX phases are not mapped to SBOM document types and they would be used as OTHER
+// this is a temporary solution until we have a better mapping
+// see: https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf
+func (u *UnserializerCDX) phaseToSBOMType(ph *cdx.LifecyclePhase) *sbom.DocumentType_SBOMType {
```

Comment:
I question if maybe we want this switch case to be under `pkg/sbom/functions.go` ?